jepsen/src/jepsen/core.clj

```diff
@@ -215,7 +265,8 @@
       (with-thread-name "jepsen nemesis"
         (info "Nemesis starting")
         (loop []
-          (when-let [op (generator/op gen test :nemesis)]
+          (when-let [op (try (generator/op gen test :nemesis)
+                             (catch Exception e e))]
```

Comment:
Same here. Perhaps it's worth moving exception logging & validation (e.g. ops are maps with these keys) into their own dedicated functions in jepsen.generator?